election_2020/intro.md

```diff
@@ -0,0 +1,34 @@
+## Setup
+
+### Virtual Environment
```

Comment:
The main README has virtual env setup, so we can refer someone there and simplify this file.